examples/generalized_linear_models/GLM-truncated-censored-regression.ipynb

```diff
@@ -0,0 +1,1356 @@
+{
```

Comment:
<p>Maybe you can add some pointers to useful resources, for people who are interested?</p><p>And also maybe a high level idea of what you would do to get better estimates here on your truncated regression -- and what you mean by "a number of factors"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>I've added text about the factors influencing the degree of bias.</p><p> </p><p>Yet to add some pointers to further reading...</p>

Reply:
<p>I've now added some info at the end of the notebook on further topics and further reading</p>